ext/hash/hash.c

```diff
@@ -1378,6 +1387,14 @@ static void php_hashcontext_dtor(zend_object *obj) {
 		efree(hash->key);
 		hash->key = NULL;
 	}
+
+	if (hash->args) {
+		if (GC_REFCOUNT(hash->args) > 1) {
+			GC_DELREF(hash->args);
+		}
+		zend_array_destroy(hash->args);
```

Comment:
The refcount manipulation + array_destroy should be zend_array_release instead.

Reply:
Done, thanks.